docs/spec.md

```diff
@@ -538,6 +561,19 @@ All integer types explicitly specified (uint32, uint64, etc.) in this section ar
 | _encoding length_ | char[]  | encoding, same encoding that was used for the request |
 | remaining bytes   | uint8[] | response payload                                      |
 
+### Asset
+
+- Response to a [fetch asset](#fetch-asset) request
+- Only supported if the server previously declared the `assets` [capability](#server-info).
+
+| Bytes           | Type             | Description                                                                                                                                                   |
+| --------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1               | opcode           | 0x04                                                                                                                                                          |
+| 4               | uint32           | request id, as given in the corresponding [request](#fetch-asset)                                                                                             |
+| 1               | uint8            | status, value is `1` if the asset was fetched succesfully, all other values are error codes                                                                   |
```

Comment:
should we make this a proper enum and say that `0` is success, `1` is error, and >1 are reserved values? (or 0=error and 1=success or something?) Then we could expand to different types of responses later without separate op codes.

Reply:
OK I will do that. 

> (or 0=error and 1=success or something?)

I prefer this, since an uninitialized value would cause an error